packages/lesswrong/components/comments/CommentsNode.tsx

```diff
@@ -34,6 +34,32 @@ const styles = (theme: ThemeType): JssStyles => ({
   },
 })
 
+export interface CommentsNodeProps {
+  treeOptions: CommentTreeOptions,
+  comment: CommentsList & {gapIndicator?: boolean},
+  startThreadTruncated?: boolean,
+  truncated?: boolean,
+  shortform?: any,
+  nestingLevel?: number,
+  expandAllThreads?:boolean,
+  expandByDefault?: boolean, // this determines whether this specific comment is expanded, without passing that expanded state to child comments
```

Comment:
Since this is a comment on an interface field, you can make it appear as a jsdoc comment
```suggestion
  /**
   * this determines whether this specific comment is expanded, without passing that 
   * expanded state to child comments
   */
  expandByDefault?: boolean,
```

Reply:
Ah, I see this was a copy paste, sorry to give you grief for a copy-paste.